src/renderers/dom/shared/utils/__tests__/getNodeForCharacterOffset-test.js

```diff
@@ -21,6 +19,17 @@ function createNode(html) {
   return node;
 }
 
+function getTestDocument(markup) {
```

Comment:
I think it's no big deal that we duplicate this. Actually convenient that you directly see what's going on now.

Reply:
I'm not sure I understand the motivation here, or why it's advantageous to copy/paste this boilerplate into several tests.